model/textparse/protobufparse.go

```diff
@@ -571,7 +577,7 @@ func (p *ProtobufParser) onSeriesOrHistogramUpdate() error {
 			return err
 		}
 	} else {
-		p.builder.Add(labels.MetricName, p.getMagicName())
+		p.builder.Add(labels.MetricName, name)
```

Comment:
Can we remove `Add(name, value string)` method from `scratchBuilder` interface since we no longer use it?